packages/client/hmi-client/src/components/workflow/ops/model-edit/tera-model-edit.vue

```diff
@@ -299,18 +299,19 @@ function updateCodeState(code: string = codeText.value, hasCodeRun: boolean = tr
 
 const createOutput = async (modelToSave: Model) => {
 	// If it's the original model, use that otherwise create a new one
-	const modelData = isReadyToCreateDefaultOutput.value ? modelToSave : await createModel(modelToSave);
-	if (!modelData) return;
+	const projectAsset = isReadyToCreateDefaultOutput.value ? modelToSave : await createModel(modelToSave);
+	if (!projectAsset) return;
```

Comment:
`createModel` must return a `model` not a `projectAsset`, if that's not the case we need to update the service.